Analysis/Tasks/PWGCF/FemtoDream/include/FemtoDream/FemtoDreamMath.h

```diff
@@ -0,0 +1,127 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file FemtoDreamMath.h
+/// \brief Definition of the FemtoDreamMath Container for math calculations of quantities related to pairs
+/// \author Valentina Mantovani Sarti, TU München, valentina.mantovani-sarti@ph.tum.de
+
+#ifndef ANALYSIS_TASKS_PWGCF_FEMTODREAM_FEMTODREAMMATH_H_
+#define ANALYSIS_TASKS_PWGCF_FEMTODREAM_FEMTODREAMMATH_H_
+
+#include "Math/Vector4D.h"
+#include "Math/Boost.h"
+#include "TLorentzVector.h"
+#include "TMath.h"
+
+#include <iostream>
+
+namespace o2::analysis
+{
+namespace femtoDream
+{
+
+/// \class FemtoDreamMath
+/// \brief Container for math calculations of quantities related to pairs
+
+class FemtoDreamMath
+{
+  public:
+
+  void setMasses(const float m1, const float m2)
```

Comment:
this is not used, so I would remove it, otherwise it is anyways not possible to use the static functions

Reply:
please remove these masses, you anyways can't use them in static functions